docs/design.md

```diff
@@ -1,6 +1,6 @@
 # Design
 
-Parca Agent implements a sampling profiler, to sample stack traces 100 times per second via eBPF. It tracks user space as well as kernel-space stack traces. From the raw data, it builds a [pprof](https://github.com/google/pprof) formatted profile and optionally sends it to a Parca server where it is stored and can be queried and analyzed over time.
+Parca Agent implements a sampling profiler, to sample stack traces 19 times per second via eBPF. It tracks user space as well as kernel-space stack traces. From the raw data, it builds a [pprof](https://github.com/google/pprof) formatted profile and optionally sends it to a Parca server where it is stored and can be queried and analyzed over time.
```

Comment:
We should open an issue at https://github.com/parca-dev/docs to sync these before the next release.

https://github.com/parca-dev/docs/blob/cfcdda347b14d776467502dc3e2695983946b2d6/Makefile#L2

Reply:
Here we go https://github.com/parca-dev/docs/issues/212.